ui/app/src/views/ViewDashboard.tsx

```diff
@@ -41,36 +41,39 @@ function ViewDashboard() {
     return generateChartsTheme('perses', muiTheme, ECHARTS_THEME_OVERRIDES);
   }, [muiTheme]);
 
-  const projectName = useParams().projectName;
-  const dashboardName = useParams().dashboardName;
+  const { projectName, dashboardName } = useParams();
+
   if (projectName === undefined || dashboardName === undefined) {
     throw new Error('Unable to get the Dashboard or Project name');
   }
-  const { data, isLoading } = useDashboard(projectName, dashboardName);
+
   const datasourceApi = useDatasourceApi();
 
-  if (!isLoading && data !== undefined) {
-    return (
-      <Box
-        component="main"
-        sx={{
-          flexGrow: 1,
-          overflow: 'hidden',
-        }}
-      >
-        <ErrorBoundary FallbackComponent={ErrorAlert}>
-          <ChartsThemeProvider themeName="perses" chartsTheme={chartsTheme}>
-            <PluginRegistry getInstalledPlugins={getInstalledPlugins} importPluginModule={importPluginModule}>
-              <ErrorBoundary FallbackComponent={ErrorAlert}>
-                <DashboardView dashboardResource={data} datasourceApi={datasourceApi} />;
-              </ErrorBoundary>
-            </PluginRegistry>
-          </ChartsThemeProvider>
-        </ErrorBoundary>
-      </Box>
-    );
-  }
-  return null;
+  const { data, isLoading } = useDashboard(projectName, dashboardName);
```

Comment:
Pushed a commit that adjusts this loading state and now you can navigate from the dashboard listing to an individual dashboard. 

Something is still off with useDashboard results though and routing generally (back button is broken when trying to go back to the dashboard listing), but I need to log off for the day

Reply:
Thank you very much, everything is working now !

For the back button actually it's working, it's just the dashboard that is adding automatically the `?start=6h` so when you click on the back button, you can see it's removing the `?start=6h`, which is then added back automatically by the dashboard.

Reply:
Ah okay gotcha, I can look into fixing this in follow up PR